ci-operator/templates/openshift/installer/cluster-launch-installer-upi-src.yaml

```diff
@@ -575,7 +576,13 @@ objects:
 
         # begin bootstrapping
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
-          RHCOS_AMI=ami-014ce8846db8b463d  # FIXME: assumes AWS_REGION is us-east-1
+          case "${BRANCH}" in  # FIXME: assumes AWS_REGION is us-east-1
+          4.1) RHCOS_AMI=ami-046fe691f52a953f9;;
+          4.2) RHCOS_AMI=ami-01e7fdcb66157b224;;
+          4.3) RHCOS_AMI=ami-014ce8846db8b463d;;
+          *) RHCOS_AMI=ami-014ce8846db8b463d;;
```

Comment:
This is OK but I would prefer automating https://github.com/openshift/installer/pull/2092#issuecomment-522663307